janitor/utils.py

```diff
@@ -411,61 +411,47 @@ def _wrapped(x, *args, **kwargs):
 def _check_instance(entry: Dict):
     """
     Function to check instances in the expand_grid function.
-    This checks if entry is a dictionary,
-    checks the instance of value in key:value pairs in entry,
-    and makes changes to other types as deemed necessary.
-    Additionally, type-specific errors are raised if unsupported data types
-    are passed in as values in the entry dictionary.
+    This checks if entry is a dictionary, checks the instance
+    of value in key:value pairs in entry, and makes changes to
+    other types as deemed necessary.
```

Comment:
I _think_ that if we adhere to SEMBR, the original should be the more appropriate version.